paratest.md

```diff
@@ -0,0 +1,71 @@
+Introduction
+============
+
+ParaTest is PHPUnit Extension allowing to run unit test in parallel, it strongly reduces unit test time consuming.
+To run test with paratest, you will need to download ParaTest package through composer and follow the different setup steps.
+
+Installation
+============
+
+1) Add ParaTest package
+
+To install with composer add the following to your `composer.json` file:
+```js
+"require": {
+    "brianium/paratest": "dev-master"
```

Comment:
I think its better to just say ``composer require brianium/paratest`` .. composer will then pick a sane default version